File: JanperCab/5-JanperCab/ClientApp/src/app/_models/duraform-order/DuraformQuoteDto.ts
import { DuraformFormDto } from './DuraformFormDto';
import { QuoteStatus } from 'src/app/_enums/QuoteStatus';
import { DuraformOrderTypeKey } from 'src/app/_enums/DuraformOrderTypeKey';

export class DuraformQuoteDto extends DuraformFormDto {
  quoteNumber: number;
  quoteStatus: QuoteStatus;

  constructor() {
    super();
    this.orderType = DuraformOrderTypeKey.Quote;
  }
}

File: JanperCab/3-Application/Dtos/DuraformQuote/DuraformQuoteDto.cs
using _1_Domain;
using _3_Application.Dtos.DuraformForm;

namespace _3_Application.Dtos.DuraformQuote
{
    public class DuraformQuoteDto : DuraformFormDto
    {
        public int QuoteNumber { get; set; }

        public QuoteStatus QuoteStatus { get; set; }
    }
}